CDCgov/prime-reportstream
# 4243 Action and item lineage indices must be larger than 0

This PR adjusts the code to use indices > 0 for items within the reports.

Test Steps:
1. Change to the `master` branch, run RS, disable the SFTP container and run the smoke tests.  Stop the smoke tests half way.  This will add some transient records for testing that the change does not affect records already in the database.
2. Change to this PR branch, re-enable the SFTP container, run RS and run the smoke tests.
3. Wait until the smoke tests complete successfully.
4. Verify that there are no exceptions related to action history or lineage.

## Changes
- Reduced info logs
- CSV serializer uses index > 0
- HL7 serializer uses index > 0 and provides it to the processing of values
- Ping test from CLI now outputs errors when it fails
- Minor Lint warning fixes

## Checklist

### Testing

### Process

## Fixes
- #issue - List GitHub issues this PR fixes

## To Be Done
*Create GitHub issues to track the work remaining, if any*
- #issue

## Specific Security-related subjects a reviewer should pay specific attention to
- Does this PR introduce new endpoints?
    - new endpoint A
    - new endpoint B
- Does this PR include changes in authentication and/or authorization of existing endpoints?
- Does this change introduce new dependencies that need vetting?
- Does this change require changes to our infrastructure?
- Does logging contain sensitive data?
- Does this PR include or remove any sensitive information itself?

If you answered '_yes_' to any of the questions above, conduct a detailed Review that addresses at least:

- What are the potential security threats and mitigations? Please list the _STRIDE_ threats and how they are mitigated
    - **S**poofing (faking authenticity)
        - Threat _T_, which could be achieved by _A_, is mitigated by _M_
    - **T**ampering (influence or sabotage the integrity of information, data, or system)
    - **R**epudiation (the ability to dispute the origin or originator of an action)
    - **I**nformation disclosure (data made available to entities who should not have it)
    - **D**enial of service (make a resource unavailable)
    - **E**levation of Privilege (reduce restrictions that apply or gain privileges one should not have)
- Have you ensured logging does not contain sensitive data?
- Have you received any additional approvals needed for this change?
